<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="selectorBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Waypoint</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Buoy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gate - No Vision</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gate - Vision</string>
    </property>
   </item>
  </widget>
  <widget class="QListView" name="taskList">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>281</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="listLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Task List</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="taskListScrollBar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>21</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Mission</string>
   </property>
  </widget>
  <widget class="QPushButton" name="receiveButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Receive Missions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Mission</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="findFileEditor">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="findFileLabel">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>60</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Mission File Path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findFileButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
